draft-ietf-quic-http.md

```diff
@@ -2174,7 +2174,9 @@ NO_ERROR (0x0):
 : HTTP_NO_ERROR in {{http-error-codes}}.
 
 PROTOCOL_ERROR (0x1):
-: No single mapping.  See new HTTP_MALFORMED_FRAME error codes defined in
+: No single mapping. Split into a few error codes, including
```

Comment:
```suggestion
: No single mapping. Split into multiple error codes, including
```

Reply:
I would say that "This is mapped to HTTP_GENERAL_PROTOCOL_ERROR except in cases where more specific error codes have been defined.  This includes HTTP_MALFORMED_FRAME, ...

Reply:
thanks, I applied your suggestion @martinthomson